aspnetcore/fundamentals/index.md

```diff
@@ -71,9 +71,9 @@ On startup, an ASP.NET Core app builds a *host*. The host encapsulates all of th
 
 There are three different hosts:
 
-* .NET WebApplication Host
-* .NET Generic Host
-* ASP.NET Core Web Host
+* [.NET WebApplication Host](xref:migration/50-to-60#new-hosting-model), also known as the minimal host.
```

Comment:
It would be "Minimum Host" not "minimum host". They are all treated as proper names in the article.